DEVELOPER_GUIDE.md

```diff
@@ -32,6 +33,15 @@ OpenSearch Go Client builds using [Go](https://golang.org/doc/install) 1.11 at a
 
 [Docker](https://docs.docker.com/install/) is required for building some OpenSearch artifacts and executing integration tests.
 
+#### Windows
+
+[WSL2](https://learn.microsoft.com/en-us/windows/wsl/install) is compatibility layer for running Linux applications on Windows.
```

Comment:
Missing "the", but let's say "To build the project on Windows, use WSL2, the compatibility ...".

Reply:
> The first phrase needs some english fixing :) Thanks for hanging in here with me!
I appreciate that you are patient with me and I am glad to work together with you!